Source/Mothership/TMoth.cpp

```diff
@@ -647,51 +695,66 @@ void* TMoth::Twig(void* par)
             DebugPrint("Message received from a Device\n");
             parent->recv(recv_buf);
             
-            uint32_t* device = static_cast<uint32_t*>(p_recv_buf); // get the first word, which will be a device address
-            if (!(*device & P_SUP_MASK)) // bound for an external?
+            P_Msg_t* msg = static_cast<P_Msg_t*>(p_recv_buf);
+            P_Msg_Hdr_t* hdr = &(msg->header);      //static_cast<P_Msg_Hdr_t*>(p_recv_buf);
+            
+            /*

```

Comment:
Still useful?

Reply:
The code certainly is for when you need to debug some pesky message receive bug. I didn't quite know what to do with it as it has no place in production but I did not want to re-write it, so I left it commented out for now.